definitions/redux-form_v5.x.x/flow_>=v0.22.1/redux-form_v5.x.x.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+import React from 'react';
+
+declare module 'redux-form' {
+  declare type InputProps = {
+    name: string,
+    value: string | number | boolean,
+    valid: boolean,
+    invalid: boolean,
+    dirty: boolean,
+    pristine: boolean,
+    active: boolean,
+    touched: boolean,
+    viisted: boolean,
+    autofiller: boolean
+  };
+  declare type FormProps = {
+    active: string,
+    asyncValidate: Function,
```

Comment:
Per the docs, it looks like this type is:

`(values: Object, dispatch: Function, props: Object) => Promise<void>`. 

Maybe we can be more specific?